tests/integration/data/kfp_1.7_stable_generic_install.yaml.j2

```diff
@@ -0,0 +1,198 @@
+bundle: kubernetes
+applications:
+  argo-controller:
+    charm: argo-controller
+    channel: 3.3/stable
+    revision: 236
+    series: focal
+    resources:
+      oci-image: 185
+    scale: 1
+    constraints: arch=amd64
+  argo-server:
+    charm: argo-server
+    channel: 3.3/stable
+    revision: 185
+    series: focal
+    resources:
+      oci-image: 186
+    scale: 1
+    constraints: arch=amd64
+  dex-auth:
```

Comment:
(Ignore this if there's more to come that'll benefit from these dependencies)

Looking at the tests now, I don't see enough value to warrant including dex-auth, istio, and oidc in these tests.  I see that we do get to test logging in with an authenticated session (`auth_session`, etc), but that feels like something we still need to test at the kubeflow-bundle level and something that generally wont break because of a change in kfp.  Outside that, I'm missing what else it adds to the test outcomes

Reply:
Yes dex, oidc, and istio are there for authentication purposes only, this is to be able to connect the KFP Client to the KFP Server. If you have an idea on how to do this differently, let me know.

Reply:
The other option I considered was [this](https://www.kubeflow.org/docs/components/pipelines/v1/sdk/connect-api/#full-kubeflow-subfrom-inside-clustersub), which uses `PodDefaults` for mounting the `ServiceAccount` volume on all pods this test suite would use, but to me it was a bit more useful in cases where we use the REST API.

Reply:
I'll continue the conversation in [this thread](https://github.com/canonical/kfp-operators/pull/238#discussion_r1252260460) as I think they're basically the same.

Reply:
This conversation does not apply anymore to the current testing environment. Feel free to re-open if needed.